### ISSUE TITLE
"Network request failed" from remoteStorage (5apps.com)

### ISSUE THREAD

**Issue by toniwiki** [Laverna/laverna#357]
I testing Laverna with remoteStorage sync on 5apps.com. I can authenticate to my account on Laverna but the remoteStorage logo always show "offline". In my Firefox console windows i can see:

~~~
[Sync] Error Object { originalError: "Network request failed.", message: "Sync failed: Network request failed.", stack: "https://laverna.cc/app/bower_components/remotestorage.js/release/stable/remotestorage.js" }
(remotestorage.js:476:21945)
~~~~

What can I doing wrong ?
Thanks

**Comment by wwebfor** [comment]
Sorry to hear that. It looks like it failed to send a request to 5apps. It is not clear why it is happening. We will check it later.

BTW, is 5apps.com available for you? Do you use a desktop version or a web version?

**Comment by toniwiki** [comment]
Yes, 5apps.com is available for me. I have test desktop and web version with same problem. From Linux.

**Comment by woodneko** [comment]
I have the same problem with the private remotestorage, I find if I use root user the connection can be set
